odl/set/__init__.py

```diff
@@ -15,119 +15,9 @@
-|Name                  |Description                                    |
-|                      |                                               |
-+======================+===============================================+
-|`NTuplesBase`         |Abstract base class for sets of n-tuples of    |
-|                      |various types                                  |
-+----------------------+-----------------------------------------------+
-|`Ntuples`             |Set of n-tuples of almost arbitrary type       |
-+----------------------+-----------------------------------------------+
-|`FnBase`              |Abstract base class for spaces of n-tuples over|
-|                      |a field                                        |
-+----------------------+-----------------------------------------------+
-|`Fn`                  |Space of n-tuples over a field allowing any    |
-|                      |scalar data type                               |
-+----------------------+-----------------------------------------------+
-|`Cn`                  |Space of n-tuples of complex numbers           |
-+----------------------+-----------------------------------------------+
-|`Rn`                  |Space of n-tuples of real numbers              |
-+----------------------+-----------------------------------------------+
-
-:math:`R^n` type spaces, CUDA implementation (module 'cu_ntuples')
-------------------------------------------------------------------
-
-Requires the compiled extension 'odlpp'
-
-+----------------------+-----------------------------------------------+
-|Name                  |Description                                    |
-+======================+===============================================+
-|`CudaNtuples`         |Set of n-tuples of almost arbitrary type       |
-+----------------------+-----------------------------------------------+
-|`CudaFn`              |Space of n-tuples over a field allowing any    |
-|                      |scalar data type                               |
-+----------------------+-----------------------------------------------+
-|`CudaCn`              |(Space of n-tuples of complex numbers) (TODO)  |
-+----------------------+-----------------------------------------------+
-|`CudaRn`              |Space of n-tuples of real numbers              |
-+----------------------+-----------------------------------------------+
-
-Function spaces (module 'default')
------------------------------------
-
-+----------------------+-----------------------------------------------+
-|Name                  |Description                                    |
-+======================+===============================================+
-|`L2`                  |Square-integrable functions taking real or     |
-|                      |complex values                                 |
-+----------------------+-----------------------------------------------+
-"""
+# TODO: write an introduction
```

Comment:
Sure. Btw, now that we have a proper API doc, the tables are not really that useful any longer. I'd rather write an introductory text about the contents and the usefulness of the submodule.